google-cloud-spanner/pom.xml

```diff
@@ -117,6 +117,12 @@
   </build>
 
   <dependencies>
+
+    <dependency>
+      <groupId>com.google.cloud</groupId>
+      <artifactId>grpc-gcp</artifactId>
+      <version>1.0.0</version>
```

Comment:
We tend to use the versions specified in [java-shared-dependencies](https://github.com/googleapis/java-shared-dependencies/blob/master/pom.xml). By that, I mean we usually do NOT specify version dependencies (like [here](https://github.com/googleapis/java-spanner/blob/master/google-cloud-spanner/pom.xml#L120-L123)), but instead rely on the versions provided by the [POM import](https://github.com/googleapis/java-spanner/blob/master/pom.xml#L107-L113) of that repository.

Could we do the same for this artifact (if it is part of any BOMs specified in the shared dependencies)?

Reply:
It is not a part of any BOMs. If I add it to the java-shared-dependencies will all the projects depending on it automatically get grpc-gcp as a dependency or they must explicitly add it to their dependencies?

Reply:
They must explicitly add it to their dependencies, but they won't need to specify a version. This makes sure that we are always using compatible library versions.